docs/app/Examples/modules/Dropdown/Types/DropdownExampleSearchDropdown.js

```diff
@@ -4,15 +4,9 @@ import { Dropdown } from 'semantic-ui-react'
 import { languageOptions } from '../common'
 // languageOptions = [ { key: 'Arabic', text: 'Arabic', value: 'Arabic' }, ...  ]
 
-/**
- * NOTE: In this example, the dropdown should contain a label on the
- * left of the text:
- * <i class="world icon"></i>
- *
- * @returns {Element}
- */
 const DropdownExampleSearchDropdown = () => (
-  <Dropdown text='Select Language' search floating labeled button className='icon' options={languageOptions} />
+  <Dropdown text='Select Language' 
+  search floating labeled button className='icon' icon='world' options={languageOptions} />
```

Comment:
If props need to be on separate lines, one prop per line please.  The closing tag should also be on its own line.

```jsx
<Dropdown
  text='Select Language'
  search
  floating
  labeled
  button
  className='icon'
  icon='world'
  options={languageOptions}
/>
```